apps/test/unit/code-studio/peer_reviews/peerReviewSubmissionsTest.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import PeerReviewSubmissions from '@cdo/apps/code-studio/peer_reviews/PeerReviewSubmissions';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import _ from 'lodash';
+
+describe("", () => {
+  let server;
+  let debounceStub;
+  let peerReviewSubmissions;
+
+  const fakePeerReviewData = [
+    {
+      submitter: 'User 1',
+      course_name: 'Course 1',
+      review_ids: [[1, 'accepted'], [2, 'accepted']]
+    },
+    {
+      submitter: 'User 2',
+      course_name: 'Course 2',
+      review_ids: [[3, 'accepted'], [4, 'accepted']]
+    }
+  ];
+
+  before(() => {
+    // stub out debounce to return the original function, so it's called immediately
+    debounceStub = sinon.stub(_, 'debounce').callsFake(f => f);
+
+    server = sinon.fakeServer.create();
+    server.respondWith("GET", '/api/v1/peer_review_submissions/index?filter=all&email=&plc_course_id=',
+      [200, { "Content-Type": "application/json" }, JSON.stringify(fakePeerReviewData)]
+    );
+
+    peerReviewSubmissions = mount(
+      <PeerReviewSubmissions
+        filterType="all"
+        courseList={[['course_1', 1], ['course_2', 2]]}
+      />
+    );
+
+    expect(server.requests.length).to.equal(1);
+    expect(server.requests[0].url).to.equal('/api/v1/peer_review_submissions/index?filter=all&email=&plc_course_id=');
+
+    server.respond();
+  });
+
+  after(() => {
+    debounceStub.restore();
+    server.restore();
+  });
+
+  beforeEach(() => {
+    server = sinon.fakeServer.create();
+  });
+
+  it("Initial call", () => {
```

Comment:
nit: word this so it reads naturally, and ideally explains a bit more about what it's testing, e.g. `it("has the expected initial course options")`